backend/accounts.go

```diff
@@ -634,6 +638,7 @@ func (backend *Backend) initPersistedAccounts() {
 	persistedAccounts := backend.config.AccountsConfig()
 outer:
 	for _, account := range backend.filterAccounts(&persistedAccounts, keystoreConnected) {
+		account := account
```

Comment:
Is this needed to make a deep copy of `account` to be passed to `createAndAddAccount`?

Reply:
It's not a deep copy. Account is a pointer.

I did it fearing the old pitfall of Go where loop vars are reused. See https://go.dev/play/p/nX3LCI2yqLR.

Some more detail: https://medium.com/swlh/use-pointer-of-for-range-loop-variable-in-go-3d3481f7ffc9

But it seems if the var itself is a pointer there is no issue? https://go.dev/play/p/6Ej6pvOJ3Ci Can't say I understand it. One of the worst pitfalls of Go, and one that is planned to be fixed afaik.

Reply:
Yes, I think we could avoid that. Until we don't use the address of the iteration variable (`&account`) everything should be fine.